src/hc32f460/Makefile

```diff
@@ -0,0 +1,38 @@
+# hc32f460 build rules
+
+# Setup the toolchain
+CROSS_PREFIX=arm-none-eabi-
+
+dirs-y += src/hc32f460 src/generic lib/hc32f460/driver/src lib/hc32f460/mcu/common
+
+CFLAGS += -mthumb -mcpu=cortex-m4 -Isrc/hc32f460 -Ilib/hc32f460/driver/inc -Ilib/hc32f460/mcu/common -Ilib/cmsis-core -DHC32F460
+
+CFLAGS_klipper.elf += --specs=nano.specs --specs=nosys.specs
+CFLAGS_klipper.elf += -T $(OUT)src/generic/armcm_link.ld
+$(OUT)klipper.elf: $(OUT)src/generic/armcm_link.ld
+
+# Add source files
+src-y += hc32f460/main.c
+src-y += hc32f460/interrupts.c
+src-y += hc32f460/gpio.c
+src-y += ../lib/hc32f460/mcu/common/system_hc32f460.c
+src-y += ../lib/hc32f460/driver/src/hc32f460_clk.c
+src-y += ../lib/hc32f460/driver/src/hc32f460_efm.c
+src-y += ../lib/hc32f460/driver/src/hc32f460_sram.c
+src-y += ../lib/hc32f460/driver/src/hc32f460_utility.c
+src-y += ../lib/hc32f460/driver/src/hc32f460_gpio.c
+src-y += ../lib/hc32f460/driver/src/hc32f460_pwc.c
+src-$(CONFIG_HAVE_GPIO_I2C) += hc32f460/i2c.c../lib/hc32f460/driver/src/hc32f460_i2c.c
```

Comment:
This looks like dead code.  (There is no i2c.c file nor is i2c enabled in Kconfig.)  To avoid confusion, best to remove these lines until i2c support is ready.

Reply:
Thanks - I removed that